lib/manager/jsonnet-bundler/artifacts.ts

```diff
@@ -0,0 +1,91 @@
+import { quote } from 'shlex';
+import { TEMPORARY_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import { readLocalFile } from '../../util/fs';
+import { getRepoStatus } from '../../util/git';
+import { regEx } from '../../util/regex';
+import {
+  PackageDependency,
+  UpdateArtifact,
+  UpdateArtifactsResult,
+} from '../types';
+
+function dependencyUrl(dep: PackageDependency): string {
+  const url = dep.lookupName;
+  if (dep.managerData.subdir) {
```

Comment:
why not add `subdir` to `lookupName`? Have you tried it?

Reply:
Tried it just now, it doesn't resolve the dependency properly:
```
DEBUG: packageFiles with updates (repository=andrein/renovate-test)
       "config": {
         "jsonnet-bundler": [
           {
             "packageFile": "jsonnet-bundler/jsonnetfile.json",
             "deps": [
               {
                 "depName": "thanos",
                 "lookupName": "https://github.com/thanos-io/thanos.git/mixin",
                 "currentValue": "v0.22.0",
                 "datasource": "git-tags",
                 "depIndex": 0,
                 "updates": [],
                 "warnings": [
                   {
                     "topic": "thanos",
                     "message": "Failed to look up dependency thanos"
                   }
                 ],
                 "versioning": "semver"
               }
             ]
           }
         ],
```